### PR TITLE
Support package:// URDF and SRDF URIs

### PR DESCRIPTION
**Pull request by mkoval**
This addresses #8.

### CODE REVIEW

**Comment by mkoval** [comment]
This is entirely backwards-compatible, so I am merging it.

File: src/urdf_loader.cpp [hunk 1]
```diff
@@ -208,11 +208,22 @@ std::string resolveURI(const std::string &path)
...
+        RAVELOG_WARN("Cannot handle this type of URI.\n");
         return "";
     }
 }
 
+std::string resolveURIorPath(const std::string &path)
+{
+    using boost::algorithm::starts_with;
+
+    if (starts_with(path, "package://") || starts_with(path, "package://")) {
```
Comment (cdellin) [inline src/urdf_loader.cpp]: I think you can just call starts_with once at this point?

Comment (mkoval) [inline src/urdf_loader.cpp]: Good catch. The second one should be file://. :-)

Comment (mkoval) [inline src/urdf_loader.cpp]: Fixed in 2d641f5c2cbb4860d6b2ce8e41e97239133147fd.

**Pull request merged**